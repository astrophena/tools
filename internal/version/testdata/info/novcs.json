{
  "name": "starlet",
  "version": "unknown",
  "commit": "",
  "built_at": "",
  "go": "go1.22.3",
  "os": "android",
  "arch": "arm64"
}
