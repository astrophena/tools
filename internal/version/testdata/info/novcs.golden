starlet unknown
built with go1.22.3, android/arm64
